src/board/v3/error.c

```diff
@@ -30,6 +30,16 @@
  * System low-level error handling and reporting
  * \{ */
 
+#define FAULT_HANDLER_SCREAMING_DEATH(msg_str)                                \
```

Comment:
Right, but you can't get the `LR` that we want that way. You'd just get the address of the fault handler itself.